### PR TITLE
docs: Update docs to reflect new pub sdk support.

### PR DESCRIPTION
**Pull request by qchong**
Fixes https://github.com/flutter/flutter/issues/6023

@abarth ptal.

Staged preview:
https://flutter-io-staging-69a8a.firebaseapp.com/upgrading/

### CODE REVIEW

**Comment by abarth** [comment]
LGTM

File: upgrading.md [hunk 1]
```diff
@@ -8,6 +8,29 @@ permalink: /upgrading/
 We recommend tracking the `master` branch in the flutter repository, which
 updates continuously as we improve Flutter.
 
+## Specifying the Flutter SDK for your project
+
+You specify dependencies on the Flutter SDK in the `pubspec.yaml` file. For
```
Comment (sethladd) [inline upgrading.md]: s/on/from

Comment (qchong) [inline upgrading.md]: fixed

**Comment by sethladd** [comment]
Thanks!

**Pull request merged**